### PR TITLE
Update danielLevitas.md

### PR DESCRIPTION
**Pull request by dlevitas**

### CODE REVIEW

File: content/team/danielLevitas.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 ---
-title: "Dan Levitas"
-date: 2021-01-20T00:40:57Z
+title: "Daniel Levitas"
+date: 2021-01-25T17:32:40Z
+draft: false
 position : 'Graduate Student'
-description : "More about Dan Levitas"
-image : '/images/avatar/franco.jpg'
+description : "Dan is a Psychology Ph.D. candidate at Indiana University. He is interested in affective salience processing, working under the direction of Dr. Thomas James. Dan is also one of the primary developers of the ezBIDS project.
```
Comment (francopestilli) [inline content/team/danielLevitas.md]: I would change the last sentence to: "Dan is the primary developer of ezBIDS."

**Review by francopestilli — CHANGES_REQUESTED** [review]

File: content/team/danielLevitas.md (hunk 1 above)
Comment (dlevitas) [reply content/team/danielLevitas.md]: okay

**Review by francopestilli — APPROVED** [review]

**Comment by francopestilli** [comment]
I think I edited your Bio and Info I am closing this. You are welcome to reopen if you need specific info from this pull. @dlevitas

**Comment by soichih** [comment]
Please edit the ohbm2022 branch - in content/team.md

**Pull request closed without merging**